spec/models/public_body_spec.rb

```diff
@@ -192,6 +192,42 @@
 
   end
 
+  describe ".internal_admin_body" do
+
+    before(:each) do
+      InfoRequest.destroy_all
+      PublicBody.destroy_all
+    end
+
+    it "creates the internal_admin_body if it didn't exist" do
+      iab = PublicBody.internal_admin_body
+      expect(iab).not_to be_nil
+    end
+
+    it "repairs the internal_admin_body if the default locale has changed" do
+      iab = PublicBody.internal_admin_body
+      expect(iab).not_to be_nil
+
+      with_default_locale(:es) do
+        I18n.with_locale(:es) do
+          found_iab = PublicBody.internal_admin_body
+          expect(found_iab).to eq(iab)
```

Comment:
you're right - it should demonstrate more clearly that:

a) initially `iab.translations.map(&:locale).include(:es)` is false
b) `iab.translations.map(&:locale).include(:es)` is true
c) (as current) it's the same internal_admin_body we first thought of (or at least that there's only one of them, we haven't created a fresh one in error)